src/main/java/uk/gov/justice/hmpps/prison/api/resource/OffenderResource.java

```diff
@@ -962,6 +964,61 @@ public List<OffenderIdentifier> getAllOffenderIdentifiers(@PathVariable("offende
         return offenderIdentifierService.getOffenderIdentifiers(prisonerNumber, includeAliases);
     }
 
+    @ApiResponses({
+        @ApiResponse(responseCode = "200", description = "OK"),
+        @ApiResponse(responseCode = "400", description = "Invalid request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
+        @ApiResponse(responseCode = "403", description = "Forbidden - user not authorised to view identifiers.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
+        @ApiResponse(responseCode = "404", description = "Requested resource not found.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
+        @ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})
+    })
+    @PreAuthorize("hasAnyRole('VIEW_PRISONER_DATA', 'GLOBAL_SEARCH')")
+    @Operation(summary = "Get a single identifier for the prisoner by sequence")
+    @GetMapping("/{offenderNo}/offender-identifiers/{offenderIdSeq}")
+    public OffenderIdentifier getOffenderIdentifier(@PathVariable("offenderNo") @Parameter(description = "The prisoner number", required = true) final String prisonerNumber,
+                                                    @PathVariable("offenderIdSeq") @Parameter(description = "The identifier sequence", required = true) final Long offenderIdSeq) {
+
+        return offenderIdentifierService.getOffenderIdentifier(prisonerNumber, offenderIdSeq);
+    }
+
+    @ApiResponses({
+        @ApiResponse(responseCode = "201", description = "Identifiers added."),
+        @ApiResponse(responseCode = "400", description = "Invalid request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
+        @ApiResponse(responseCode = "403", description = "Forbidden - user not authorised to add identifiers.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
+        @ApiResponse(responseCode = "404", description = "Prisoner not found.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
+        @ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})
+    })
+    @VerifyOffenderAccess
```

Comment:
I'm not sure we want this in place? Won't this break when trying to update a restricted patient's identifiers for example?